### DISCUSSION TITLE
Remove Obsolete Invitations

### DISCUSSION THREAD

**Discussion by juaneloDev** [outline/outline#3839] [Self Hosting] [answered]
Should add ability to selectively delete users who have not accepted Outline invites. Should prevent them from appearing in list of users to add to groups, for example.

**Comment by tommoor** [comment, marked as answer]
Settings -> Members -> Filter by "Invited" -> Revoke Invite…

**Comment by mtukus** [comment]
Not on the self hosted version?

<img width="1409" height="530" alt="image" src="https://github.com/user-attachments/assets/3671d7e7-6b32-4f8e-a753-b6f96bce64bc" />

**Reply by tommoor** [reply]
You would need to be an admin to manage invitations, I'm guessing that's the problem here.

**Reply by mtukus** [reply]
Yeah, true! I assigned myself admin rights manually through the DB and then it started working. Thanks for replying anyways!

Note: You can also see pages like integrations with Notion when you are admin, while when I was not admin I was not able to see a lot of pages. Good to know for others that are using Outline.
